readFile.js

```diff
@@ -0,0 +1,34 @@
+const fileSystem = require('fs');
+
+let finalString = '';
```

Comment:
Multiple calls to `readFile` will append the contents into finalString, so it shouldn't be a module level scope variable.

example:
```javascript
readFile('file-a').then(() => readFile('file-b'))
```

will resolve to `file-a` contents concatenated with `file-b` contents, but it should resolve to the contents of `file-b`